src/views/RequestRepository/index.tsx

```diff
@@ -9,10 +9,6 @@ import classnames from 'classnames';
 import { DateTime } from 'luxon';
```

Comment:
Since this isn't really a `view` anymore, this file/component can be moved to `src/components/`